src/components/BSelect/BSelect.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <div>
+    <div>
+      <h4>{{ selectLabel }} <span v-if=isRequired>*</span></h4>
+    </div>
+    <div 
+      v-bind:aria-expanded="opened"
+      v-bind:aria-owns="id"
+      role="combobox"
+      tabindex="0"
+    >
+      <ul
+        v-bind:aria-label="selectLabel"
+        v-bind:class="getUlClass()"
+        role="listbox"
+        v-bind:id="id"
+      >
+        <li 
+          v-bind:class="getLiClass()" 
+          v-if=!selectedOption
+          v-on:click="openList"
+          role="option"
+          style="color: #cccccc"
```

Comment:
Some styles are in the style tag, some are not. Why are some not?

Reply:
This specific case was just changing the coloring from grey to black if a selection has been made.  Would you prefer that I did it in a different way?